docs/blog/newsletter-august-2017.rst

```diff
@@ -0,0 +1,84 @@
+
+In other news, Slack's been humming away as usual this month, with so many excellent conversations that it's getting harder and harder to choose just a few to cover here. We've also got nine different local meetups happening around the world this month. So even if you can't join us for the conferences, don't miss out on your other opportunities to interact with the community!
+
+Oh, and also, as I mentioned last month, we're looking to expand our little editorial team for the newsletter. Special thanks to Claire Lundeby (@claire.lundeby), who'll be joining the crew this month! We're still looking for one or two more folks who already spend a lot of time on the Write the Docs Slack, and want to help us turn insightful Slack conversations into mini-features. Drop me a message on Slack @kobrien042, if that sounds like you!
+
+In the meantime, let's see what people were talking about this month!
+
+************************************************
+What resume advice is the *right* resume advice?
+************************************************
+
+For anyone currently job-hunting, this story will probably be pretty resonant. The question arose, this month, of how to navigate the virtual *ocean* of resume guidance, tips, and templates out there, and decide which advice is the best advice. Since most of us have been in this position before, there were lots of helpful suggestions:
+
+* There is no 'right way' to write a resume. All of the mechanical and organization rules are dependant on you, your background, the job listing, the target company, etc.
+* Treat your resume like any other kind of documentation – start by asking who your reader is, and what they need from your resume. Align your content to the themes and priorities present in their job listing/any other company materials you can get your hands on. Try a `keyword density tool <http://tools.seobook.com/general/keyword-density/)>`_ for help identifying those themes.
+* Don't limit your resume to your paying work. Include personal and community projects – stuff you work on in your free time or as a side hustle. Just because you're not getting paid, doesn't mean you're not gaining experience. (XP > GP)
+* Make sure you've got your supporting materials in hand. Craft a tailored cover letter for every job. (No form letters!) Get your LinkedIn profile in shape. Spruce up your portfolio; even if you're just starting out, write up some samples for your favorite tool or process.
+* Apply to the right jobs. Job hunting takes a HUGE amount of effort. Better to invest that time into perfecting a few really exciting applications, than putting minimum effort into dozens of applications you don't really care about.
```

Comment:
Is it "apply to" or "apply for"? Something looks strange but I don't trust myself totally on this one...